libpdbg/libpdbg.h

```diff
@@ -50,10 +50,10 @@ struct pdbg_target_class;
  * @brief Identifies the processor type
  */
 enum pdbg_proc {
-	PDBG_PROC_UNKNOWN,  /**< Unknown processor */
-	PDBG_PROC_P8,       /**< POWER8 processor  */
-	PDBG_PROC_P9,       /**< POWER9 processor  */
-	PDBG_PROC_P10,      /**< POWER10 processor */
+  PDBG_PROC_UNKNOWN, /**< Unknown processor */
```

Comment:
applying clang-format should be a separate commit, that way only the functional changes will be part of this commit.

Reply:
> applying clang-format should be a separate commit, that way only the functional changes will be part of this commit.

Agreed. Didn't imagine that PDBG has never been gone through clang in its lifetime otherwise I wouldn't have done that :-(